spec/javascripts/components/input-spec.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-env jasmine, jquery */
+/* global GOVUK */
+
+describe('Input', function () {
+  var element
+
+  beforeEach(function () {
+    element = $(
```

Comment:
Is there no way around using jQuery here? Looking in other js tests, they use an approach akin to:

```suggestion
    element = document.createElement('div')
    element.innerHTML =
      '<div class="gem-c-input--reveal-password" data-module="input" data-show="Show" data-hide="Hide" data-password="password" data-announce-show="Your password is shown" data-announce-hide="Your password is hidden">' +
        '<input name="password" class="gem-c-input govuk-input" id="input" type="password">' +
      '</div>'
    
```

Reply:
It's okay to use jquery in tests, because the jasmine suite has its own version of jquery, so even after we remove all jquery from our code we can still have it in tests. Makes writing some of our tests a bit easier, too.